Fix li.operation div.content overflow

Murmand opened the pull request:
When a description of an operation was too long, it overflows box without possibility to scroll the content (on page .../swagger/ui/index.html). This pull request fix the problem and add poissibity to scroll x-overflow if it overflows.

fehguy commented:
I'm not seeing the issue without your fix.  Can you please share a screenshot?

![image](https://cloud.githubusercontent.com/assets/249413/20586323/d920ab4a-b1b7-11e6-9678-d65fd95aff80.png)

Murmand commented:
Sure, sorry about bad descripted pull request.

Before pull request:
![before](https://cloud.githubusercontent.com/assets/6674990/20589254/a306c97c-b21b-11e6-8e49-0efc39e376db.png)

After pull request:
![after](https://cloud.githubusercontent.com/assets/6674990/20589258/ab53de3a-b21b-11e6-8b08-8d85f31a0e27.png)

fehguy commented:
You mean you have a single parameter description that is so long that it causes this?  I don't think we want a scroll bar at the bottom, maybe a more reasonable description is the solution.

Murmand commented:
If you think that would be a solution... :-) I added the pull request because the scroll bar shows only if the content overflows, otherwise the scrollbar is hidden. If you think that the right solution is to use specific description without long words, sure, it is a solution, but I am not sure if it is a right solution :-) You are author of this useful and great project, so this is your decision. I have already made my one (that this is the right way) before I added the pull request :-)

webron commented:
Apologies for the (very) delayed feedback.

Unfortunately, our code base might have deviated from the time this PR was submitted, and we cannot easily merge it (certainly not against `master`), and so, we're going to close this PR.

If you find the change important and would like to resubmit it, please let us know. Be aware that we are focused on the 3.X version at the moment (now `master`), and new changes should be applied to it.

We appreciate the time you've taken into submitting this PR. We're now making extra effort to review PRs within a few days at most, to avoid situations like this.
